packages/angular/plugins/component-testing.ts

```diff
@@ -172,11 +172,14 @@ function normalizeBuildTargetOptions(
   buildOptions.index =
     typeof buildOptions.index === 'string'
       ? joinPathFragments(offset, buildOptions.index)
-      : (buildOptions.index.input = joinPathFragments(
-          offset,
-          buildOptions.index.input
-        ));
-  buildOptions.tsConfig = joinPathFragments(offset, buildOptions.tsConfig);
+      : {
+          ...buildOptions.index,
+          input: joinPathFragments(offset, buildOptions.index.input),
+        };
+  // cypress creates a tsconfig if one isn't preset
+  // that contains all the support required for angular and component tests
+  delete buildOptions.tsConfig;
```

Comment:
I had thought about making a tmp tsconfig that is a combination of the `tsconfig.cy.json` and `buildOptions.tsConfig` but just seems simpler to use what was working before with cypress making the tsconfig.